### PR TITLE
Add spanish translations

### PR DESCRIPTION
**Pull request by larzuk91**

### CODE REVIEW

File: src/Resources/translations/ResetPasswordBundle.es.xlf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="1">
+                <source>%count% año|%count% years</source>
```
Comment (bocharsky-bw) [inline src/Resources/translations/ResetPasswordBundle.es.xlf:6]:
```suggestion
                <source>%count% year|%count% years</source>
```

😉

**Review by bocharsky-bw — CHANGES_REQUESTED** [review]
Ha, I can review in Spanish! Easy translation, well-known words :p 

P.S. One more repo, please! 😇  https://github.com/SymfonyCasts/verify-email-bundle

File: src/Resources/translations/ResetPasswordBundle.es.xlf (hunk 1 above)
Comment (larzuk91) [reply src/Resources/translations/ResetPasswordBundle.es.xlf:6]: Wooooooooppps!!

**Comment by larzuk91** [comment]
Ha! Nice one Victor, your Spanish it's better than mine now!

**Review by bocharsky-bw — APPROVED** [review]
👍

**Review by jrushlow — APPROVED** [review]
Thanks @larzuk91

**Comment by weaverryan** [comment]
Thanks Mr Diego!

**Pull request closed without merging**